### PR TITLE
Update BANNO.M2MTRANSFERS.V3.POW

### PR DESCRIPTION
**Pull request by tkainz**
When the saved external account record is expired or deleted, changing the name displayed for existing transfers to that account from the share description/nickname to 'xAABCCCC' where 'A' is the last 2 digits of the member number, 'B' is 'S' or 'L' for share/loan and then, 'C', the Share/Loan ID.

### CODE REVIEW

**Review by jucarson — APPROVED** [review]
Changes look good Tom.

**Comment by tkainz** [comment]
Goooood morning @jordan-keenan can you please put this peer review on your agenda for today?  Many, many thanks!

**Review by jordan-keenan — APPROVED** [review]
@tkainz , Code looks good!
Can you change the following spelling on line 2114 while you are making this change? Correct the spelling of editible to editable:
"[* Determine if existing transfer is editible or not
*]"

**Review by jordan-keenan — COMMENTED** [review]
Thanks @tkainz. Changes look good!

File: m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW [hunk 1]
```diff
@@ -2155,42 +2159,15 @@ PROCEDURE BUILDSLTRANSFERLIST
 
          TRANRECIPACCTID(TRANCOUNT)=SHARE TRANSFER:ID
          TRANRECIPNICKNAME(TRANCOUNT)=RECIPNICKNAME(EXTRECFOUND)
-[* If the recipient nickname is blank (not saved) then get the [SHARE/LOAN]:NICKNAME if available else
-** the [SHARE/LOAN]:DESCRIPTION
+[* Use 'x'+last 2 of member number+'L' or 'S'+S/L ID format instead of description/nickname
```
[the remark this answers is not in the document]
Comment (jordan-keenan) [reply m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW:2162]: Thanks @tkainz

**Pull request merged**